### ISSUE TITLE
Text Analysis Idea

### ISSUE THREAD

**Issue by kammerdienerb** [jclar168/MiniProject1#1]
For my idea, I'd like to do the word frequency analysis on two historical articles from Wikipedia. Then I'd like to compare one of those articles to a scientific arcticle (maybe something in biology). It might be interesting to see the difference in verb tense between the two categories.

**Comment by jclar168** [comment]
An analysis comparing Wikipedia articles seems like a good project, although if the conclusion of the verb tense comparison is what you would expect (biology mainly uses present tense while history mainly uses past tense) then it might be good to add something else to make the analysis more interesting.

**Comment by kammerdienerb** [comment]
Perhaps I could further apply the idea to two scientific articles, but from two scientifc topics that are of vastly different ages (i.e. one old, very established field, and one emerging topic).

**Comment by jclar168** [comment]
Yes, I think that could be very interesting. From that you could get information like the kind of language used to describe developing fields vs. established ones, or, depending on the difference in age of the articles, the difference in academic writing style over the years.
